ios/Mattermost.xcodeproj/project.pbxproj

```diff
@@ -891,6 +892,20 @@
 			shellScript = "diff \"${PODS_PODFILE_DIR_PATH}/Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [ $? != 0 ] ; then\n    # print error to STDERR\n    echo \"error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\" >&2\n    exit 1\nfi\n# This output is used by Xcode 'outputs' to avoid re-running this script phase.\necho \"SUCCESS\" > \"${SCRIPT_OUTPUT_FILE_0}\"\n";
 			showEnvVarsInLog = 0;
 		};
+		64399EEDB3C64470AC8AD28F /* Upload Debug Symbols to Sentry */ = {
```

Comment:
Wasn't this phase already defined?

Reply:
Just double checked it and it seems that it got duplicated.  Removing it now.